app/src/components/layout/AppBarBottom.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div
+    class="flex w-full justify-center border-t-1 border-gray-200 dark:border-gray-700 bg-gray-100 dark:bg-dark-400 dark:text-gray-300"
```

Comment:
@anbraten The background color in light mode should be the same as the popups. More consistent and looks better IMHO.